VideoApi/VideoApi.Services/AudioPlatformServiceStub.cs

```diff
@@ -121,15 +121,7 @@ public async Task<WowzaGetStreamRecorderResponse> GetAudioStreamInfoAsync(Guid h
 
         public async Task<AudioPlatformServiceResponse> DeleteAudioStreamAsync(Guid hearingId)
```

Comment:
doesnt differ from DeleteAudioApplicationAsync why do we need it twice?

Reply:
The stub is implemented in exactly the same way. It's been the case for a while and Rider has been pointing it out for a long time. Maybe a question for the original author